charts/rancher-gke-operator/1.0.300/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+name: rancher-gke-operator
+description: A Helm chart for provisioning GKE clusters
+home: https://github.com/rancher/gke-operator
+sources:
+  - "https://github.com/rancher/gke-operator"
+version: 1.0.300
```

Comment:
This is already live ? And not supposed to be here according to: https://github.com/rancher/rancher/issues/33528
https://github.com/rancher/system-charts/tree/release-v2.5/charts/rancher-gke-operator/1.0.300

Reply:
Can you try pulling in/rebasing release to this branch to see if it goes away.

Reply:
I created a PR with this rebase: https://github.com/rancher/system-charts/pull/544